test/ng/filter/orderBySpec.js

```diff
@@ -309,6 +309,14 @@ describe('Filter: orderBy', function() {
 
         expect(orderBy(items, expr)).toEqual(sorted);
       });
+
+      it('should treat `null` values bigger then all other values except `undefined`', function() {
+        var items = [undefined, null, 'z', {}, 999, false];
+        var expr = null;
+        var sorted = [false, 999, {}, 'z', null, undefined];
+
+        expect(orderBy(items, expr)).toEqual(sorted);
```

Comment:
Just to be sure, could you also add an expectation for sorting in descending order?